src/components/SearchInput/index.jsx

```diff
@@ -85,42 +114,41 @@ function SearchInput() {
       <p className="mt-4 text-base font-semibold">
         Search the video using Haystack!
       </p>
-      <form
-        className="flex flex-col w-96 mt-4 border-2 rounded-lg border-red-500"
-        onSubmit={handleSubmit}
-      >
+      <div className="flex flex-col w-96 h-10 mt-4 border-2 rounded-lg border-red-500">
         <input
-          className="pl-2 rounded-md border-red-500 outline-none"
+          className="flex h-full pl-2 rounded-md border-red-500 outline-none"
           type="text"
           placeholder="Haystack 검색"
           value={userInput}
           onChange={handleUserInputChange}
-          onKeyDown={handleArrowKeyPress}
-          ref={inputRef}
+          onKeyDown={handleKeyPress}
           autoFocus
         />
-        {userInput && (
-          <div>
-            <button
-              className={`text-left absolute w-96 rounded-md ${autoCompletions.length ? "border-red-500  bg-white border-2" : ""} mt-1`}
-              type="button"
-              onClick={handleAutoCompletionClick}
-            >
-              {autoCompletions.map((element, index) => (
-                <p
+        {autoCompletions.length > 0 && showAutoCompletions && (
+          <button
+            className="absolute w-96 mt-10 rounded-md border-red-500 bg-white border-2 text-left"
```

Comment:
css 속성 순서가 변경되면 좋을 것 같습니다!
```suggestion
            className="absolute w-96 mt-10 border-2 rounded-md border-red-500 bg-white text-left"
```